pkgs/by-name/mc/mcat/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "mcat";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "Skardyy";
+    repo = "mcat";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+wpcLf94FJhkufytKgVnRonivH9CVY/X0+bXx3Z8HKI=";
+  };
+
+  cargoHash = "sha256-pi/nts8YqXms5ps6bt+KQRh8twvk5qfkh5FF+/0tMOE=";
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Cat command for documents / images / videos and more";
+    homepage = "https://github.com/Skardyy/mcat";
+    changelog = "https://github.com/Skardyy/mcat/releases/tag/v${finalAttrs.version}";
```

the release page doesn't seem to contain the actual changelog...
```suggestion
    changelog = "https://github.com/Skardyy/mcat/blob/v${finalAttrs.version}/CHANNELOG.md";
```
and don't forget to fix the typo when updating to the next version